save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkContestProjectController.kt

```diff
@@ -61,6 +62,25 @@ class LnkContestProjectController(
         lnkContestProjectService.getByContestName(contestName)
     )
         .map {
-            ContestResult(it.project.name, it.project.organization.name, it.score)
+            it.toContestResult()
+        }
+
+    /**
+     * @param organizationName
+     * @param projectName
+     * @param numberOfContests
+     * @return best [numberOfContests] contests of a project with name [projectName]
+     */
+    @GetMapping("/{organizationName}/{projectName}/best")
+    @PreAuthorize("permitAll()")
```

Comment:
How will we handle results for private projects?

Reply:
The point is that contest info is public for everybody no matter if it is private or public

Reply:
Feels really weird that we hide private projects from everyone, but they can be linked to public contests and be uncovered.. Need to think more carefully about it and create a warning screen for private projects entering a public contest. (in a separate issue, of course)

Reply:
Mentioned in #900